data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg

```diff
@@ -500,7 +500,7 @@
 
         [message]
             speaker=Tallin
-            message= _ "Stand firm, men. The dwarven defenders of Knalga rise again. Now it remains to be seen whether or not they recognize us as friends."
+            message= _ "So it is true! The dwarven defenders of Knalga have risen once again. Stand firm men! Now it remains to be seen whether or not they recognize us as friends."
```

Comment:
Please add a comment for the translators, as they'll likely see this line without the context of the already-translated line before it, and "risen" can be ambiguous. Something line (on the previous line)
```
# po: The dwarves are skeletal undead
```